#!/usr/bin/env python
PACKAGE = "move_by_ft_wrist"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("rate", int_t, 0, "node rate",    10, 0,   100)

gen.add("fx", double_t, 0, "force x ",    0.0, -100.0,   100.0)
gen.add("fy", double_t, 0, "force y ",    0.0, -100.0,   100.0)
gen.add("fz", double_t, 0, "force z ",    0.0, -100.0,   100.0)

gen.add("tx", double_t, 0, "torque x ",    0.0, -10.0,   10.0)
gen.add("ty", double_t, 0, "torque y ",    0.0, -10.0,   10.0)
gen.add("tz", double_t, 0, "torque z ",    0.0, -10.0,   10.0)

gen.add("frame_id", str_t, 0, "frame_id for goals", "arm_right_tool_link")
gen.add("topic_name", str_t, 0, "topic for the wrenchstamped", "/right_wrist_ft_fake")


exit(gen.generate(PACKAGE, PACKAGE, "ForceTorque"))